### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by Ranjith-K-07**
This is the front end application initial commit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+  "name": "sign_page",
```
Comment (teamlead-rdl) [inline package.json:2]: Change the text from sign_page to ResearchAndDesignLabs

File: build/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (teamlead-rdl) [inline build/robots.txt:1]: Delete this file

**Review by teamlead-rdl — CHANGES_REQUESTED** [review]
Please make those changes

File: package.json (hunk 1 above)
Comment (Ranjith-K-07) [reply package.json]: Taken Care

File: build/robots.txt (hunk 2 above)
Comment (Ranjith-K-07) [reply build/robots.txt:1]: As agreed up on not deleting

**Review by teamlead-rdl — APPROVED** [review]

**Pull request merged**